classes/blocks/class-articles.php

```diff
@@ -179,6 +181,12 @@ public function prepare_data( $fields ): array {
 			$recent_posts = $this->populate_post_items( $sliced_posts );
 		}
 
+		// Enqueue js for the frontend.
+		if ( ! $this->is_rest_request() ) {
+			wp_enqueue_script( 'load-more', P4GBKS_PLUGIN_URL . 'public/js/load_more.js', [ 'jquery' ], '0.3', true );
+			wp_localize_script( 'load-more', 'more_url', '/wp-admin/admin-ajax.php' );
```

Comment:
Here the 3rd argument of `wp_localize_script ()` function expects array.

Reply:
@sagarsdeshmukh good catch! I amended the commit.